pages/common/progress.md

```diff
@@ -0,0 +1,12 @@
+# progress
+
+> Show progress of an ongoing system command. Supported commands include: cp, mv, dd, tar, bsdtar, cat, rsync, scp, grep, fgrep, egrep, cut, sort, md5sum, sha1sum, sha224sum, sha256sum, sha384sum, sha512sum, adb, gzip, gunzip, bzip2, bunzip2, xz, unxz, lzma, unlzma, 7z, 7za, zip, unzip. 
```

Comment:
```suggestion
> Show progress of an ongoing system command.
```

I don't think this will work with the linter + it looks messy. However good idea with it.